tests/parsing/parchmint-parser.test.js

```diff
@@ -773,6 +899,123 @@ const duplicateIDParchmintComponents = '"components": [\n' +
         '    }\n' +
         ']';
 
+const validParchmintConnections = '"connections": [\n' +
+        '    {\n' +
+        '        "id": "unique-mixer-flow-connection-id",\n' +
+        '        "name": "mixer-flow-connection",\n' +
+        '        "layer": "unique-flow-layer-id-string",\n' +
+        '        "source": {\n' +
+        '            "component": "unique-mixer-id-string",\n' +
+        '            "port": "output-port"\n' +
+        '        },\n' +
+        '        "sinks": [\n' +
+        '            {\n' +
+        '               "component": "unique-mixer-id-string",\n' +
+        '               "port": "input-port"\n' +
+        '            }\n' +
+        '        ]\n' +
+        '    }\n' +
+        ']';
```

Comment:
This doesn't have to go into this PR but I find these constructed JSON slug strings very annoying. Would it be possible to use template literals from ES6 (https://stackoverflow.com/questions/8963693/how-to-create-json-string-in-javascript) or to create a JSON object and then use `.stringify()` to convert it to the needed string? I feel like either of those would be cleaner than having all the `+` and `\n` for the string construction.

Reply:
This will be fixed by #62.